exercises/beer-song/src/test/scala/BeerSongTest.scala

```diff
@@ -0,0 +1,60 @@
+import org.scalatest._
+
+class BeerSongTest extends FunSuite {
+
+  test("first generic verse") {
+    val expected = "99 bottles of beer on the wall, 99 bottles of beer.\nTake one down and pass it around, 98 bottles of beer on the wall.\n"
+    val beerSong = new BeerSong()
```

Comment:
There is no internal state. So it makes more sense to have an `object` here instead of a `class`. Just:
```scala
assert(expected == Beersong.verse(99))
```